README.rst

```diff
@@ -29,11 +29,52 @@ The following features are available:
 
 -   Automatic *OCIO* **Reference** configuration generation for *aces-dev*
     *CTL* reference implementation.
--   Configurable generator producing the *OCIO* **Studio** configuration.
+
+    - Discovery of *aces-dev* *CTL* transforms.
+    - Generation of the *CTL* transforms graph.
+
+-   Configurable generator producing the *OCIO* **CG** and **Studio**
+    configurations.
+-   Builtins *CLF* transforms along with generator and discovery support.
 
```

Comment:
Using "Builtin" here is potentially confusing since OCIO has a Built-in Transform feature that we are using in the configs but the CLF format does not actually support that.  You could write something like "Includes CLF transforms along with ...".

It might also be helpful to casual observers if you add a quick note describing the release state of each component.  Basically that the Reference config is essentially done, the CG config is close, and the Studio config is still in planning.

Reply:
Yep, will do!